_drafts/oryx/technology.md

```diff
@@ -0,0 +1,31 @@
+# Oryx technologies
+
+Oryx is a frontend framework that is designed to help developers build web applications using modern technologies.
+It is built on a variety of technologies and in this article, we will take a closer look at each of these technologies and see how they contribute to the development of Oryx.
+
+## HTML
+HTML is the backbone of the web. It is the markup language used to create web pages and applications. In Oryx, HTML is used to structure the content and layout of the application. It provides a semantic structure to the page, making it easier to understand and maintain.
+
+## CSS
+CSS is a style sheet language used to describe the visual presentation of a document written in HTML. In Oryx, CSS is used to style the components and layout of the application. It allows developers to create a consistent visual style across the application and ensure that it is responsive and accessible.
+
+## JavaScript
+JavaScript is a high-level programming language that is widely used in web development. In Oryx, JavaScript is used to add interactivity and dynamic behavior to the application. It allows developers to create complex logic and manipulate the DOM in real-time.
+
+## TypeScript
+TypeScript is a superset of JavaScript that adds optional static typing, classes, and interfaces to the language. In Oryx, TypeScript is used to provide type safety and better code organization. It allows developers to catch errors early in the development process and provides better code navigation and refactoring capabilities.
+
+## Web Components
+Web Components are a set of web platform APIs that allow developers to create custom, reusable, and encapsulated HTML elements. In Oryx, Web Components are used to create reusable UI components that can be used across the application. It allows developers to create complex components with minimal code duplication and provides better code organization and maintainability. At the same time it means Oryx component are compatible with any other frontend framework and can be integrated into almost any web-application.
+
+## Lit
+Lit is a lightweight library for creating web components using TypeScript and HTML templates. In Oryx, Lit is used to create reusable and encapsulated UI components that can be easily styled and extended. It provides a simple and intuitive API for creating custom elements and allows developers to write less boilerplate code.
+
+## RxJS
+RxJS is a reactive programming library for JavaScript. In Oryx, RxJS is used to handle asynchronous operations. It provides a declarative way to handle events and data streams, making it easier to manage complex data flows and state transitions.
```

Comment:
We could add that 
- RxJS is used in the service layer of Oryx to provide a reactive system that continuees to emit new values while you navigate through the (SPA) experience. 
- RxJS is not a main citizen in the component logic, as we try to hide the reactivity altogether to avoid complexity